packages/client/hmi-client/src/components/models/Model.vue

```diff
@@ -480,6 +480,20 @@ const editorKeyHandler = (event: KeyboardEvent) => {
 			renderer.render();
 		}
 	}
+	if (event.key === 'Enter' && renderer) {
+		if (renderer?.nodeSelection) {
+			renderer.deselectNode(renderer.nodeSelection);
+			renderer.nodeSelection
+				.selectAll('.no-drag')
+				.style('opacity', 0)
+				.style('visibility', 'hidden');
+			renderer.nodeSelection = null;
+		}
+		if (renderer?.edgeSelection) {
+			renderer.deselectEdge(renderer.edgeSelection);
+			renderer.edgeSelection = null;
+		}
+	}
```

Comment:
If you are already checking `if (renderer && ..)` you shouldn't need to use the option-chaining thingy. I think you can just do directly
```
if (renderer.nodeSelection) { ... }
```

Reply:
Totally correct thanks!